test/operator/setup-testdata.sh

```diff
@@ -18,22 +31,23 @@ function setup() {
   mkdir -p "$OUTPUT_DIR"
   cp "${DIR}/testdata/configs/latest/imageset-config.yaml" "${OUTPUT_DIR}/"
   find "$DATA_DIR" -type f -exec sed -i -E 's@REGISTRY_CATALOGNAMESPACE@'"$REGISTRY_CATALOGNAMESPACE"'@g' {} \;
+
+  create_buildx_builder
 }
 
 function build_push_bundles() {
   echo -e "\nBuilding and pushing bundle images"
   for d in `find "${DATA_DIR}" -maxdepth 1 -name *-bundle-*`; do
     local img="${REGISTRY}/$(basename $d | cut -d- -f1)-operator/$(basename $d | cut -d- -f1-2):$(basename $d | cut -d- -f3)"
     pushd $d
-    docker build -t $img -f bundle.Dockerfile .
-    docker push $img
+    docker buildx build --push -t $img -f bundle.Dockerfile .
```

Comment:
Side note: you'll need to install and enable [`docker buildx`](https://docs.docker.com/buildx/working-with-buildx/) to run e2e tests locally now if it isn't already available on your machine.